packages/create-block/lib/templates.js

```diff
@@ -65,27 +72,42 @@ const templates = {
 	},
 };
 
-const getTemplate = ( templateName ) => {
-	if ( ! templates[ templateName ] ) {
+const getTemplate = async ( templateName ) => {
+
+	const isCoreTemplate = await checkIsCoreTemplate( templateName );
+	if ( isCoreTemplate ) {
+		return templates[ templateName ];
+	}
```

Comment:
It feels like it can be simplified to:
```suggestion
	if ( templates[ templateName ] ) {
		return templates[ templateName ];
	}
```
We can rename `templates` to `coreTemplates` if you think it would make the code easier to follow.

Reply:
Simplified and renamed  👍